clusterware-customize/lib/functions/customize-repository-s3.functions.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
```

Comment:
Probably doesn't need a `#!` line as this is a source file rather than an executed file.